docs/docs/features/command-line-interface.md

```diff
@@ -30,6 +32,16 @@ NOTE: if you previously installed the legacy CLI, you will need to uninstall it
 npm uninstall -g immich
 ```
 
+## Installation (Docker)
+
+If npm is not available on your system you can try the Docker version
+
+```bash
+docker run -it -v "$(pwd)":/import:ro -e IMMICH_API_KEY=https://your-immich-instance/api -e IMMICH_API_KEY=your-api-key ghcr.io/immich-app/immich-cli:latest
```

Comment:
The environment variable for the server should be IMMICH_INSTANCE_URL

Reply:
This has already been fixed in a separate pr